h/static/styles/core/_color.scss

```diff
@@ -20,6 +20,22 @@ $brand: #d00032;
 $highlight: #58cef4;
 $group-organization-color: #6d676d;
 
+$success-bg: #d4edda;
+$success-border: #c3e6cb;
+
+$error-bg: #fff3cd;
+$error-border: #ffeeba;
```

Comment:
We don't have any standard colours in h that could be used here. I copied these from [Bootstrap's success and warning alert boxes](https://getbootstrap.com/docs/4.4/components/alerts/). They can be changed in a follow-up PR if we want to